task-tracker-webapp/src/main/java/com/isa/tasktrackerwebapp/service/TaskService.java

```diff
@@ -0,0 +1,62 @@
+package com.isa.tasktrackerwebapp.service;
+
+import com.isa.tasktrackerwebapp.model.Task;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public interface TaskService {
```

Comment:
dlaczego to jest interfejs? to powinna być klasa, która implementuje ewentualnie interfejs; w interfejsie też nie trzeba pisać słowa public, bo to jest domyślny podyfikator tutaj